test/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -41,13 +39,13 @@ import org.codice.compliance.utils.decorators.IdpPostResponseDecorator
 import org.codice.security.saml.SamlProtocol.Binding.HTTP_POST
 import org.codice.security.sign.Decoder
 
-class PostBindingVerifier(private val response: IdpPostResponseDecorator) {
+class PostBindingVerifier(private val response: IdpPostResponseDecorator) : BindingVerifier() {
```

Comment:
❓ What's the advantage to subclassing `BindingVerifier` here? We get no behaviors/polymorphism from it; we only use a static method from its companion.

Reply:
It was just so we could reuse the functionality. And we figure the reusable functionality that we need between POST and Redirect will grow.